src/inactive-blog-filter.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * Returns whether or not the blog should be marked as inactive
+ * Criteria for filtering (in milliseconds) is based on filters.json under blog.inactive
+ */
+const fs = require('fs');
+const dotenv = require('dotenv');
```

Comment:
Use our `config` module from `src/config`, which will do this once.

Reply:
Oh, that's right! I'll fix this.